### PR TITLE
Two minor copy edits

### PR DESCRIPTION
**Pull request by mccambridge**
Great read. Thanks for putting this out there! I noticed a couple of tiny grammar issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Denote [working hours](https://support.google.com/calendar/answer/83117?hl=en#wo
...
+Give constructive, not critical feedback<sup>[3](#footnotes)</sup>. Feedback is negatively critical when it surfaces something wrong with someone or something they produced, especially without any mention of ways to make their behavior or their product better. Critical feedback on work often looks like "you don't write enough tests" or "your code quality isn't good enough". Personal criticism can be more severe and often looks like "you should be less judgemental" or "you are a burden because you ask too many questions”. Constructive feedback is more about how a person can can do better rather than what they are doing wrong. If you want someone to do something better, you should tell them what better looks like. Ask a question to get a discussion rolling, to gain context, and then if you see room for improvement give declarative feedback to that effect. This creates an environment where people understand what success looks like instead of just feeling like they are unsuccessful.
```
Comment (mccambridge) [inline README.md:32]: Last sentence: s/feel/feeling

Comment (brianarn) [inline README.md:32]: Thanks for clarifying! In the rich view, it was much clearer, but this note helps.

**Comment by brianarn** [comment]
Seems reasonable to me. :+1: 

Pinging @jennazee to get thoughts and maybe merge. :)

File: README.md (hunk 1 above)
Comment (mccambridge) [inline README.md:32]: haha. yeah, i saw how long that line was and thought you might have to hunt for the change

**Comment by jennazee** [comment]
neato. thanks for that!

**Pull request merged**